docs/input.md

```diff
@@ -323,3 +323,21 @@ d_log_temp_tol = 1e-8              # Tolerance in cooling table between subseque
 Finally, a more comprehensive descriptions of various conventions used for cooling
 functions (and the chosen implementation in AthenaPK) can be found in [Cooling Notes](cooling_notes.md)
 and a notebook comparing various cooling tables (and their conversion) in [cooling/cooling.ipynb](cooling/cooling.ipynb).
+
+## Boundary conditions
+
+In addition to enrolling custom boundary conditions, three general options are currently supported by default:
+
+- `periodic`: work without restrictions (implemented in upstream Parthenon)
+- `outflow`: work without restrictions (implemented in upstream Parthenon)
```

Comment:
Is this a constant extrapolation from the last valid cell into the ghost cells? I think it would help to say explicitly what this is.

Reply:
Yes, that's it. I expanded the comment.
I also added a warning when being used for subsonic flows.